lib/OpenQA/JobTemplates.pm

```diff
@@ -21,12 +21,53 @@ use Exporter 'import';
 use Carp;
 use Try::Tiny;
 use JSON::Validator;
+use YAML::PP 0.020;
 
 our $VERSION   = '0.0.1';
 our @EXPORT_OK = qw(
-  &validate_data
+  &validate_data &load_yaml &dump_yaml
 );
 
+my $YP = _init_yaml_processor();
```

Comment:
I don't mind the indirection too much, but i guess i would have just used `my $YP = YAML::PP->new(...);`

Reply:
Hm, I don't like code blocks out of subs too much...

Reply:
Both work for me i guess. 🤷‍♀